TACC/Core-Portal
# task/FP-821: Google Drive UI fixes

## Related Jira tickets: ##

* [FP-821](https://jira.tacc.utexas.edu/browse/FP-821)

## Summary of Changes: ##
-  Fix button and text formatting issues in Google Drive components
- Update system selector in copy/select modals to default to the correct system
- Allow systems to be excluded from the system selector and disable Google Drive in the Applications file selector modal.

## Testing Steps: ##
- Confirm that Google Drive can't be selected from the selector modal in Applications.
- Open the Copy modal in Google Drive and confirm that the right-hand listing matches the system in the selector dropdown.
